### DISCUSSION TITLE
DocumentTests

### DISCUSSION THREAD

**Discussion by WilliamHeleodoro** [balta-io/forum#95] [Tira Dúvidas] [answered]
Ao executar o comando dotnet test  no terminal do windows ele retorna o seguinte erro:
No test is available in C:\Vision System\C#\Carreira Desenvolvedor Backend .NET\8 - Modelando Domínios Ricos\PaymentContext\PaymentContext.Tests\bin\Debug\net6.0\PaymentContext.Tests.dll. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.

Tem algum problema por ser o visual studio 2022 ?

**Comment by WilliamHeleodoro** [comment]
![image](https://user-images.githubusercontent.com/54126120/200967487-a40a2c7c-427a-4009-99e8-24b7a9f57119.png)

**Comment by WilliamHeleodoro** [comment]
Tentei adicionar outros pacotes de testadapter, mas acontece o mesmo erro.
  
  <ItemGroup>
    <PackageReference Include="coverlet.collector" Version="3.2.0">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
    <PackageReference Include="Flunt" Version="1.0.2" />
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
    <PackageReference Include="MSTest.TestFramework" Version="1.1.18" />
    <PackageReference Include="NUnit3TestAdapter" Version="4.3.0" />
    <PackageReference Include="NUnitTestAdapter" Version="2.3.0" />
  </ItemGroup>

**Comment by WilliamHeleodoro** [comment]
![image](https://user-images.githubusercontent.com/54126120/200969625-0e61d637-818f-4c71-ade2-ba7f3ec4de11.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @WilliamHeleodoro, você marcou a classe com `TestClass`?
Me parece que ele não conseguiu encontrar nenhum teste.

**Comment by WilliamHeleodoro** [comment]
Bom Dia @andrebaltieri 
Perfeito! Acabei esquecendo esse detalhe, muito obrigado.
Tive que voltar as versões dos pacotes também. Mas deu certo.
